layouts/partials/html-head.hbs

```diff
@@ -12,6 +12,19 @@
 
   <link rel="apple-touch-icon" href="/static/apple-touch-icon.png">
   <link rel="icon" sizes="32x32" type="image/png" href="/static/favicon.png">
+  
+  <meta property="og:site_name" content="Node.js Foundation">
```

Comment:
Was trying to think of something other than just `Node.js` since that's not the name of the site either, it's the tech. Foundation seemed like the next best extension, but not perfect. I'll change to just `Node.js` for now 👍